### PR TITLE
zh-cn for Scala Tour:extractor-objects.md

### PR DESCRIPTION
**Pull request by realwunan**
Simplified Chinese version of Scala Tour: Extractor Objects

### CODE REVIEW

File: _zh-cn/tour/extractor-objects.md [hunk 1]
```diff
@@ -13,3 +13,55 @@ language: zh-cn
...
+val customer1ID = CustomerID("Sukyoung")  // Sukyoung--23098234908
+customer1ID match {
+  case CustomerID(name) => println(name)  // prints Sukyoung
+  case _ => println("Could not extract a CustomerID")
+}
+```
+
+这里 `apply` 方法用 `name` 创建一个 `CustomerID` 字符串.  而 `unapply` 方法正好相反, 它返回 `name` .  当我们调用  `CustomerID("Sukyoung")` , 其实是调用了 `CustomerID.apply("Sukyoung")` 的简化语法.  当我们调用 `case CustomerID(name) => println(name)`, 就是在调用提取器方法.
+
+因为在定义一个值的时候可以使用一个模式来引入一个新的变量,  提取器可以用来初始化这个变量,  使用 unapply 方法来生成值.
```
Comment (liufengyun) [inline _zh-cn/tour/extractor-objects.md]: `变量定义可以使用模式引入变量`?

Comment (realwunan) [reply _zh-cn/tour/extractor-objects.md]: It's much better, thanks a lot.

File: _zh-cn/tour/extractor-objects.md [hunk 2]
```diff
@@ -13,3 +13,55 @@ language: zh-cn
...
+
+```tut:fail
+val CustomerID(name3) = "-asdfasdfasdf"
+```
+
+`unapply` 方法的返回值应当符合下面的某一条: 
+
+* 如果只是用来测试,  返回一个 `Boolean` 类型的值.  例如 `case even()`.
+* 如果只是用来提取单个 T 类型的值,  返回 `Option[T]`.
+* 如果你想要提取多个值,  类型分别为 `T1,...,Tn`, 把它们放在一个可选的元祖中 `Option[(T1,...,Tn)]`.
```
[the remark this answers is not in the document]
Comment (realwunan) [reply _zh-cn/tour/extractor-objects.md]: Thanks for pointing it out.

[the remark this answers is not in the document]
Comment (hepin1989) [reply _zh-cn/tour/extractor-objects.md]: 如果...可以...

[the remark this answers is not in the document]
Comment (realwunan) [reply _zh-cn/tour/extractor-objects.md]: Thanks a lot.

**Comment by liufengyun** [comment]
LGTM, thanks a lot for the contribution @realwunan 🎉

**Pull request merged**